manual/manual.tex

```diff
@@ -133,126 +171,397 @@ \section{Quick Start}
-  from Stopify to suspend execution. Stopify supports `arguments` in full, but
-  it also supports two simple special cases that improve performance.
+The \lstinline|arguments| object makes it difficult for Stopify to resume
+execution after suspension. Stopify supports \lstinline|arguments| in full, but
+it also supports two simple special cases that improve performance.
 
-  - `simple`: use this option if the program (1) does not use `arguments` to
+\begin{itemize}
+
+  \item Use \lstinline|"simple"| if the program (1) does not use \lstinline|arguments| to
     access declared formal arguments and (2) only reads additional
-    arguments using the `arguments` object.
+    arguments using the \lstinline|arguments| object.
 
-  - `faithful`: use this option if the program (1) does not use `arguments`
+  \item Use \lstinline|"faithful"| if the program (1) does not use \lstinline|arguments|
     to access declared formal arguments and (2) may read or write additional
-    arguments using the `arguments` object.
+    arguments using the \lstinline|arguments| object.
+
+  \item Use \lstinline|"full"| for full support of JavaScript's \lstinline|arguments| object.
 
-  - `full`: use this option for full support of JavaScript's `arguments` object.
+\end{itemize}
 
-  Command-line: `--js-args` (default: `simple`)
+\subsection{Higher Order Functions (\texttt{.hofs}) \label{hofs-flag}}
 
-  JavaScript API: `jsArgs`
+Programs cannot use bulitin higher-order functions (e.g., \lstinline|.map|,
+\lstinline|.filter|, etc.) with Stopify, since Stopify cannot instrument native
+code. The \lstinline|.hofs| flag has two possible values:
 
-- **Higher-order functions.** Programs cannot use higher-order functions
-  (e.g., `.map`, `.filter`, etc.) with Stopify, since Stopify cannot
-  instrument native code.
+\begin{itemize}
 
-  - `builtin`: with this option, Stopify assumes that the program does not
-    use any native higher-order functions.
+  \item Use \lstinline|"builtin"| if the program does not use any native
+  higher-order functions.
 
-  - `fill`: with this option, Stopify rewrites programs that use native
+  \item Use \lstinline|"fill"| to have Stopify rewrite programs that use native
     higher-order functions to use polyfills written in JavaScript.
 
-  Command-line: `--hofs` (default: `builtin`)
+\end{itemize}
+
+\subsection{Getters and Setters (\texttt{.getters})\label{getters-flag}}
 
-  JavaScript API: `hofs`
+Programs that suspend execution within getters/setters incur a lot of overhead
+with Stopify. The \lstinline|.getters| flag has two possible values:
 
-- **Getters and setters**. Programs that suspend execution within
-  getters/setters incur a lot  of overhead with Stopify.
+\begin{itemize}
 
-  - `true`: with this option, Stopify instruments the program to support
-    suspension within getters/setters.
+  \item Use \lstinline|true| to have Stopify instrument the program to
```

Comment:
This is not what the `--getters` flag currently does. If only adds support for getters if the program uses `get` or `Object.defineProperty`.